src/backend/defs/rest-api/objects/Settings.yaml

```diff
@@ -1,19 +1,27 @@
 ---
 type: object
 properties:
-  default_language:
-    type: string
-  default_timezone:
-    type: string
-  date_format:
+  default_locale:
     type: string
+    default: fr_FR
```

Comment:
I was wrong, it should `fr-FR`, (this what I get from browser)

Reply:
fixed in 8ab87b3879ea6a48d62968500bafd81f50d77915